packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -82,6 +98,12 @@ function EditorCanvas( {
 				setCanvasMode( 'edit' );
 			}
 		},
+		onClickCapture: ( event ) => {
```

Comment:
@mcsf if I remember correctly you've looked at this event recently. Do you think it's fine to use?

Reply:
Yeah, this might be fine.

Indeed, I proposed using a capture-phase listener in #59563 to allow clicking on a whole row to select it. It's worth noting (and I had forgotten about this in our recent chat) that I ended up removing it in #59803 because the requirements had changed. But if the goal is to disable interactions going into the canvas, this may be good.

I expressed some of my concerns with `on*Capture` as follows:

> Mostly I think it’s important to remain aware that this is basically a lateral system. Technically it’s part of the same event propagation system, but since we never use Capture, it could lead to blind spots in our mental model, might make things trickier to debug… I’m saying all this in abstract, as potential risks, because I was the one suggesting Capture to begin with :)
>
> For instance when we debug a problem we may inspect the node from an element down to its leaves to find some problematic listener, and we forget to inspect in the other direction: looking for problems near the document root

In this particular case I'd also pay attention to the fact that we want to make the canvas "mostly" inert but not completely. What does this mean for event interception? Plus, do users and devices understand that this area is now partially disabled? Etc.

Reply:
>do users and devices understand that this area is now partially disabled? Etc.

I think so yes, since we have aria-disabled and role='button'